graph/validation_barrier.go

```diff
@@ -180,16 +249,22 @@ func (v *ValidationBarrier) WaitForDependants(job interface{}) error {
 	// Any ChannelUpdate or NodeAnnouncement jobs will need to wait on the
 	// completion of any active ChannelAnnouncement jobs related to them.
 	case *lnwire.ChannelUpdate1:
-		signals, ok = v.chanEdgeDependencies[msg.ShortChannelID]
+		annID = msg.ShortChannelID
+
+		// TODO: If ok is false, we have serious issues.
+		parentJobIDs, ok = v.jobDependencies[childJobID]
```

Comment:
why does this read not need a mutex lock?

Reply:
it's locked above no?